src/Entity/BannerItem.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baraja\Banner\Entity;
+
+
+use Baraja\Doctrine\Identifier\Identifier;
+use Baraja\Localization\TranslateObject;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity]
+#[ORM\Table(name: 'core__banner_item')]
+class BannerItem
+{
+	use Identifier;
+	use TranslateObject;
+
+	#[ORM\ManyToOne(targetEntity: Banner::class, inversedBy: 'bannerItems')]
+	private Banner $banner_id;
```

Comment:
ID si Doctrine doplní sama:

```suggestion
	private Banner $banner;
```